### PR TITLE
Cleanup IELE semantics in preparation for Solidity end-to-end tests

### PR DESCRIPTION
**Pull request by dwightguth**

### CODE REVIEW

**Comment by dwightguth** [comment]
Jenkins: test this please

**Comment by dwightguth** [comment]
Jenkins: test this please

File: ethereum.md [hunk 1]
```diff
@@ -95,7 +95,7 @@ To do so, we'll extend sort `JSON` with some IELE specific syntax, and provide a
     rule <k> startTx => #finalizeBlock ... </k>
          <txPending> .List </txPending>
 
-    rule <k> startTx => loadTx(#sender(TN, TP, TG, TT, TV, #unparseByteStack(DATA), TW, TR, TS)) ... </k>
```
Comment (theo25) [inline ethereum.md]: Do we need to keep `#sender` around in the semantics anymore?

Comment (dwightguth) [reply ethereum.md]: I guess not, but there's some other code that's not currently being used laying around, it might be better to do it in a separate PR.

Comment (theo25) [reply ethereum.md]: OK cool.

**Comment by dwightguth** [comment]
@theo25 the installation instructions were fine but when testing on a fresh vm I found a minor bug that I decided to also fix in this PR, can you take a look at the commit I just added?

**Comment by theo25** [comment]
Also looks good to me.

**Pull request merged**